components/root/root.jsx

```diff
@@ -232,6 +233,10 @@ export default class Root extends React.PureComponent {
                 GlobalActions.redirectUserToDefaultTeam();
             }
             this.onConfigLoaded();
+        }).then(() => {
```

Comment:
This is called on component did mount. What happens when the user is still not logged in?

Reply:
removed the statement from root.jsx.